sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3883,7 +3884,7 @@ _ZN2cl4sycl7contextC2ERKSt6vectorINS0_6deviceESaIS3_EERKNS0_13property_listE
 _ZN2cl4sycl7contextC2ERKSt6vectorINS0_6deviceESaIS3_EESt8functionIFvNS0_14exception_listEEERKNS0_13property_listE
 _ZN2cl4sycl7contextC2ERKSt8functionIFvNS0_14exception_listEEERKNS0_13property_listE
 _ZN2cl4sycl7contextC2ESt10shared_ptrINS0_6detail12context_implEE
-_ZN2cl4sycl7handler10mem_adviseEPKvm14_pi_mem_advice
+_ZN2cl4sycl7handler10mem_adviseEPKvmi
```

Comment:
Isn't this ABI breaking change we should avoid?

Reply:
No, this symbol was added around 16 days ago, so it's safe to break ABI inside the single release.